src/Publisher.php

```diff
@@ -52,7 +52,7 @@ public function __invoke(Update $update): string
             'retry' => $update->getRetry(),
         ];
 
-        $jwt = ($this->jwtProvider)();
+        $jwt = ($this->jwtProvider)($update->getTargets());
```

Comment:
Shouldn't we pass the whole `Update` object? It gives more flexibility to the user.

Reply:
Yes good idea I tried and update the pull request